cloudconfig/powershell_helpers.go

```diff
@@ -19,34 +28,34 @@ $ErrorActionPreference = "Stop"
 
 function ExecRetry($command, $maxRetryCount = 10, $retryInterval=2)
```

Comment:
Yup, I am aware this string is yucky, however it is outside the scope of this PR. We do have a trello card about it, should get solved when we get to it